Extensions/Spine/spineruntimeobject-pixi-renderer.ts

```diff
@@ -0,0 +1,141 @@
+namespace gdjs {
+
+  const isSpine = (obj: any): obj is pixi_spine.Spine => obj instanceof pixi_spine.Spine;
+
+  export class SpineRuntimeObjectPixiRenderer {
+    private _object: gdjs.SpineRuntimeObject;
+    private _rendererObject!: pixi_spine.Spine | PIXI.Container;
+    private _isAnimationComplete = true;
+    private _spinePropertiesReplacer: Record<string, any> = { };
+
+    /**
+     * @param runtimeObject The object to render
+     * @param instanceContainer The container in which the object is
+     */
+    constructor(
+      runtimeObject: gdjs.SpineRuntimeObject,
+      private instanceContainer: gdjs.RuntimeInstanceContainer
+    ) {
+      this._object = runtimeObject;
+
+      this.constructSpine();
```

Comment:
Refactor to `this._rendererObject = this.constructSpine();`, so you can declare _rendererObject without !